randovania/games/am2r/exporter/patch_data_factory.py

```diff
@@ -282,5 +374,5 @@ def create_data(self) -> dict:
             "game_patches": self._create_game_patches(pickup_list, item_data, self.rng),
             "door_locks": self._create_door_locks(),
             "hints": self._create_hints(self.rng),
-            "cosmetics": self._create_cosmetics()
+            "cosmetics": self._create_cosmetics(self.rng)
```

Comment:
Don't use the main RNG for cosmetic things! Create a new one with `Random(description.get_seed_for_player(players_config.player_index))`

Reply:
Out of curiosity, what's bad about using main RNG for cosmetic stuff?

Reply:
if the main RNG is used later for non-cosmetic stuff (which is a thing!), it means that cosmetic settings can create gameplay differences

Reply:
I personally prefer making a new Random object for each distinct randomized cosmetic, so that if two players have the same setting for that specific cosmetic, they get the same result regardless of any other randomized cosmetics